src/taskpane/fallbackauthtaskpane.ts

```diff
@@ -0,0 +1,72 @@
+/*
```

Comment:
The name of this file caused me a lot of confusion. This is code that's going to run in the _dialog_, not the _taskpane_.

Reply:
We could rename this - will require some plumbing work - but I can make the change

Reply:
fixed